vectorizing/test/test_vectorizer_heb.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from unittest import TestCase
+import stemming.stemming
+from vectorizing import vectorizer_heb
+from main_flow import main_flow
+from utils import presistor
+
+INSTRUCTIONS_TOP_WORDS = 'inst_top_words.pkl'
+INGREDIENTS_TOP_WORDS = 'ing_top_words.pkl'
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('vectorizingEngTest')
+vectorizer = vectorizer_heb.VectorizerHeb()
+stemmer = stemming.stemming.StemmerHebrew()
+main_flow.loadCache()
+top_instruc_dict = presistor.load_parameter_cache_from_disk('app/inst_top_words.pkl')
+top_ingred_dict = presistor.load_parameter_cache_from_disk('app/ing_top_words.pkl')
+
+# please make sure your working directory for each run is RecipeGrabber and not vectorizing/test
+
+
+class TestVectorizerHeb(TestCase):
+
+    def test_example_to_vector_heb1(self):
+        text = "היי תכינו את הבצל ממש ממש טוב ככה יופי טוב שיצא מושלם חם חם מהתנור ואז תדחפו אותו למיקרו ותוסיפו שמן מים וחצילים"
+        result = vectorizer.example_to_vector(text, top_instruc_dict, top_ingred_dict)
+        assert result == []
+
+    def test_example_to_vector_heb2(self):
+        text = "יש לי רק דבר אחד לומר לך הוא מכל הלב"
+        result = vectorizer.example_to_vector(text, top_instruc_dict, top_ingred_dict)
+        assert result == []
+
+    def test_example_to_vector_heb3(self):
+        text = " פתאום תרנגול תרתיחי מים"
+        result = vectorizer.example_to_vector(text, top_instruc_dict, top_ingred_dict)
+        assert result == []
+
+    def test_example_to_vector_heb4(self):
+        text = "i do wonder what would happen if everything was in english"
+        result = vectorizer.example_to_vector(text, top_instruc_dict, top_ingred_dict)
+        assert result == []
+
+    def test_example_to_vector_heb5(self):
+        text = "בוא ננסה רשימה אחרת של טופ וורדס"
+        test_top_instruc_dict = {'בוא': 1, 'אחר': 2}
+        test_top_ingred_dict = {'יש': 1, 'טופ': 2}
+        result = vectorizer.example_to_vector(text, test_top_instruc_dict, test_top_ingred_dict)
```

Comment:
same here the asset has to be meaningful, you can stringify the result and compare OR maybe compare specific places within the vector.